rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -325,6 +325,36 @@ public void run() {
             }
         };
         this.statsUpdater.scheduleRepeating(RMConfig.get().getStatisticsRefreshTime());
+
+        scheduleRequestMaxNumberOfNodesIn(0);
+        }
+
+    private void scheduleRequestMaxNumberOfNodesIn(int milliseconds) {
+        new Timer() {
```

Comment:
Why do you need a Timer here, especially with value `0`? is the alive nodes limit not static?